packages/near-fast-auth-signer/src/components/SignMultichain/SignMultichain.tsx

```diff
@@ -73,7 +63,7 @@ function SignMultichain() {
   });
 
   const onError = (text: string) => {
-    window.parent.postMessage({ type: 'multichainError', message: text }, '*');
+    window.parent.postMessage({ type: 'multiChain', message: text }, '*');
```

Comment:
I don't mind the change, but would it be ideal to indicate that if the passing message is error or success response? so that receiving end can apply appropriate decision around given message? (eg, if error, show red/orange colour with pop message?)

Reply:
I think we can discuss this, we can provide it on the type or the message body (e.g. making it an object with more descriptive information. 

It's the wallet that will handle this incoming message. 

It's ok to do it in another PR?